lib/2wp-utils.js

```diff
@@ -15,6 +15,8 @@ const ADDRESS_TYPES_CODES = {
 const BTC_TO_RSK_MINIMUM_CONFIRMATIONS = 3;
 const TO_BRIDGE_GAS_PRICE = 2;
 const BRIDGE_ADDRESS = '0x0000000000000000000000000000000001000006';
+const WHITELIST_CHANGE_PK = '3890187a3071327cee08467ba1b44ed4c13adb2da0d5ffcc0563c371fa88259c';
+const WHITELIST_CHANGE_ADDR = '87d2a0f33744929da08b65fd62b627ea52b25f8e';
```

Comment:
These constants are already exported from the `whitelisting.js` file. Please, import them and use them here. No need to duplicate them.
Also check that they are redeclared again in `01_03_51-2wp_release_event.js` and `01_03_52-disable_lock_whitelist.js`. Delete them from there and import them on those files from `whitelisting.js`.

Reply:
Should the method in it's entirety be moved to whitelisting.js? Or should it stay here? What do you think?